aten/src/ATen/TensorIndexing.h

```diff
@@ -495,7 +517,10 @@ static inline Tensor dispatch_index_put_(Tensor& self, std::vector<Tensor>&& ind
 // See NOTE [ Setting `disable_slice_optimization` when calling C++ tensor indexing functions from Python ]
 static inline Tensor get_item(const Tensor& self, const ArrayRef<TensorIndex>& indices, bool disable_slice_optimization = false) {
   at::Device self_device = self.device();
-  IntArrayRef self_sizes = self.sizes();
+  // TODO: nested tensor does not have a size (yet) so for now we represent its size as null
+  //       may need to be changed after we reach a better solution for nested tensor size
+  c10::optional<IntArrayRef> self_sizes = c10::nullopt;
+  if (! self.is_nested()) self_sizes = self.sizes();
```

Comment:
I think we use curly brackets  even for one 1-line statements. Alternatively I think you can also use a ternary operator (which some people might even like less).
`self.is_nested() ? c10::nullopt : self.sizes()`

Reply:
yeah I tried but I would get a compilation error:
```
../aten/src/ATen/TensorIndexing.h:453:66: error: incompatible operand types ('const c10::nullopt_t' and 'c10::IntArrayRef' (aka 'ArrayRef<long long>'))
    c10::optional<IntArrayRef> result_sizes = result.is_nested() ? c10::nullopt : result.sizes();
```
could be solved though, I think, by constructing `c10::optional<IntArrayRef>` instances from `c10::nullopt` and `result.sizes()`